RELEASE-NOTES.md

```diff
@@ -29,21 +29,50 @@
 
 ### Version 3.1.2-missing-ftpa-tab-beta
 **EUI-3825** Fix for missing FTPA tab contents.
+### Version 4.0.0
```

Comment:
Why is `Version 4.0.0` below a load of `Version 3.x.x` in here? Shouldn't `4.0.0` be at the top?

Reply:
Changed now.